pkg/image/admission/imagepolicy/imagepolicy_test.go

```diff
@@ -99,29 +98,27 @@ func TestDefaultPolicy(t *testing.T) {
 	}
 
 	client := &imageclient.Clientset{}
-	imageResp := 0
-	// respond to images in this order: goodImage, badImage
 	client.AddReactor("get", "images", func(action clientgotesting.Action) (handled bool, ret runtime.Object, err error) {
-		image := goodImage
-		if imageResp%2 == 1 {
```

Comment:
this switched from order-dependent responses to actually responding based on what was requested